app/helpers/green_lanes_helper.rb

```diff
@@ -1,10 +1,12 @@
 module GreenLanesHelper
   def exemption_checkbox_checked?(resource_id, exemption_code)
-    category_assessments_checked(resource_id)&.include?(exemption_code)
+    params.dig(:exemptions, resource_id.to_s)
+          &.include?(exemption_code)
```

Comment:
We should not use the `&`,
it is defensive, to avoid errors in case exemptions is missing,
which happen when clicking "back" link.